build.sh

```diff
@@ -279,7 +279,7 @@ elif [ "$PIPELINE" = 'update' ]; then
         "$BUILD_CONTEXT"
     date -Iseconds
 fi
-docker images
+docker images "$DOCKER_REPOSITORY"
```

Comment:
Displaying all images can be useful for knowing the build result or build environment for the diagnosis of unexpected build issues

Reply:
ok.